data/campaigns/Winds_of_Fate/scenarios/05_Threshold.cfg

```diff
@@ -224,62 +213,24 @@
         [/ai]
     [/side]
 
+    {SPAWNER_SIDE 4  "Blood Bat"       *^Ft  {ON_DIFFICULTY4 0.1  0.25 0.5  0.5}  6 1}
+    {SPAWNER_SIDE 5  "Woodland Boar"   *^Ft  {ON_DIFFICULTY4 0.1  0.25 0.5  0.5}  6 1}
+    {SPAWNER_SIDE 6  "Elder Falcon"    *^Fds {ON_DIFFICULTY4 0.1  0.25 0.5  0.5}  6 1}
+    {SPAWNER_SIDE 7  "Great Wolf"      *^Fds {ON_DIFFICULTY4 0.1  0.25 0.5  0.5}  6 1}
+    {SPAWNER_SIDE 8  "Grand Dragonfly" Ss    {ON_DIFFICULTY4 0.05 0.1  0.25 0.5}  6 1}
+    {SPAWNER_SIDE 9  "Swamp Lizard"    Ss    {ON_DIFFICULTY4 0.02 0.05 0.1  0.25} 3 1}
+    {SPAWNER_SIDE 10  "Firebane Ant"   Hd    {ON_DIFFICULTY4 0.05 0.1  0.25 0.5}  6 1}
+    {SPAWNER_SIDE 11  "Firebomb Ant"   Hd    {ON_DIFFICULTY4 0.05 0.1  0.25 0.5}  6 1}
+    {SPAWNER_SIDE 12  "Giant Spider"   Md    {ON_DIFFICULTY4 0.02 0.05 0.1  0.25} 3 1}
+
+    # Wisp
     [side]
-        side=4
-        controller=ai
-        hidden=yes
-        team_name=jungle
-        user_team_name= _ "Jungle"
-        no_leader=yes
-        defeat_condition=never
-        [ai]
-            village_value=0
-            [avoid]
-                terrain=*^V*
-            [/avoid]
-        [/ai]
-    [/side]
-
-    [side]
-        side=5
-        controller=ai
-        hidden=yes
-        team_name=forest
-        user_team_name= _ "Forest"
-        no_leader=yes
-        defeat_condition=never
-        [ai]
-            village_value=0
-            [avoid]
-                terrain=*^V*
-            [/avoid]
-        [/ai]
-    [/side]
-
-    [side]
-        side=6
+        side=13
```

Comment:
What's the reason for side 13? Just asking because `wmllint` complains about a side number being out of sequence (I note there are sides 1-5, is this supposed to be side 4?)

Reply:
Sides 4 through 12 are created by the above `SPAWNER_SIDE` macro [calls](https://github.com/wesnoth/wesnoth/blob/8cebaf5b51838d82525835dc67085ec2c9c81433/data/campaigns/Winds_of_Fate/scenarios/05_Threshold.cfg#L216-L224) as defined over [here](https://github.com/wesnoth/wesnoth/blob/8cebaf5b51838d82525835dc67085ec2c9c81433/data/campaigns/Winds_of_Fate/utils/macros.cfg#L97-L111).

I guess since this must have passed the CI checks five months ago there maybe was some update to `wmllint` since then which is causing it to complain?

Reply:
I thought it might be something like that. Maybe need to add a comment to make this clearer.

I don't think `wmllint` is actually used in CI yet. That's what #7082 is supposed to record.

Reply:
In that case you should append `# wmllint: skip-side` to every call of the `SPAWNER_SIDE` macro. (Or if the macro call is really long you could put that above each call instead.)

Reply:
Should I do that for the `SPAWNER_SIDE` calls in scenarios 1 and 9 also? They are not causing any complaints from `wmllint`, presumably because there is no explicit `[side]` definition following them with a higher side number. But maybe liberal use of `# wmllint: skip-side` is best practice in general?

---

Looks like `wmllint` has two other complaints:

`rank for mainline campaigns should be a multiple of 5 between 0 and 299 (included)`

`11_Crosswind.cfg", line 169: unknown 'land_on_keep' referred to by id`

Fixing the first involves changing the ranks for WoF, SoF and LoW. The second complaint message seems unclear to me.

Reply:
> Should I do that for the `SPAWNER_SIDE` calls in scenarios 1 and 9 also? They are not causing any complaints from `wmllint`, presumably because there is no explicit `[side]` definition following them with a higher side number. But maybe liberal use of `# wmllint: skip-side` is best practice in general?

It can't hurt, on the off-chance that someone later on decides to add sides after them.

> `11_Crosswind.cfg", line 169: unknown 'land_on_keep' referred to by id`
> 
> [...] message seems unclear to me.

Without seeing the line in context, my guess would be that it saw `id=land_on_keep` in some tag that has nothing to do with units and assumed it was supposed to refer to a unit.